Docs/reference/config.toml

```diff
@@ -1,4 +1,4 @@
-baseurl = "/mongo-csharp-driver/2.9"
+baseurl = "/mongo-csharp-driver/2.10"
```

Comment:
Choosing 2.10 over 2.9

Reply:
What is the affect of this change?

Reply:
![image](https://user-images.githubusercontent.com/33040432/66153810-2822fb80-e5ea-11e9-91f4-a2824082c618.png)

Reply:
It seems that'll force the 2.10 docs to link to 2.9 resources

Reply:
Ack